packages/session-replay-browser/src/helpers.ts

```diff
@@ -130,3 +137,17 @@ export const getServerUrl = (serverZone?: keyof typeof ServerZone): string => {
 
   return SESSION_REPLAY_SERVER_URL;
 };
+
+export const getStorageSize = async (): Promise<StorageData> => {
+  const globalScope = getGlobalScope();
+  if (globalScope) {
+    const storeEstimate: ChromeStorageEstimate = await globalScope.navigator.storage.estimate();
+    const totalStorageSize = storeEstimate.usage ? Math.round(storeEstimate.usage / KB_SIZE) : 0;
+    const percentOfQuota =
+      storeEstimate.usage && storeEstimate.quota
+        ? Math.round((storeEstimate.usage / storeEstimate.quota + Number.EPSILON) * 1000) / 1000
+        : 0;
+    return { totalStorageSize, percentOfQuota, usageDetails: JSON.stringify(storeEstimate.usageDetails) };
+  }
```

Comment:
```suggestion
    const {usage, quota, usageDetails}: ChromeStorageEstimate = await globalScope.navigator.storage.estimate();
    const totalStorageSize = usage ? Math.round(usage / KB_SIZE) : 0;
    const percentOfQuota =
      usage && quota
        ? Math.round((usage / quota + Number.EPSILON) * 1000) / 1000
        : 0;
    return { totalStorageSize, percentOfQuota, usageDetails: JSON.stringify(usageDetails) };
```

Nit: I think this is slightly cleaner?